src/main/webapp/script.js

```diff
@@ -0,0 +1,40 @@
+function authorizeCallback(authResult) {
+  if (authResult['code']) {
+    $('#signinButton').attr('style', 'display: none');
+    var xhr = new XMLHttpRequest();
```

Comment:
The codebase has been using `fetch()` for http requests. Why a new library is introduced here? I think continue using `fetch()` can make things more consistent.

Reply:
ill try it out -- i used this library because this is what the official google documentation used